app/src/main/java/org/fossasia/openevent/app/common/data/repository/EventRepository.java

```diff
@@ -94,4 +99,19 @@ public EventRepository(IUtilModel utilModel, IDatabaseRepository databaseReposit
             .build();
     }
 
+    @NonNull
+    @Override
+    public Observable<Event> updateEvent(Event event) {
+        if (!utilModel.isConnected()) {
+            return Observable.error(new Throwable(Constants.NO_NETWORK));
+        }
+
+        event.setTickets(null);
```

Comment:
>Also, there is no indicator in Event list denoting the draft events

New group is added as Draft. All the draft events are shown at last under draft header.